src/ucs/sys/sys.h

```diff
@@ -150,13 +150,14 @@ uint64_t ucs_generate_uuid(uint64_t seed);
  *   - stderr
  *
  * *p_fstream is filled with the stream handle, *p_need_close is set to whether
- * fclose() should be called to release resources, *p_next_token to the remainder
- * of config_str.
+ * fclose() should be called to release resources (1) or not (0), *p_next_token
+ * is set to the remainder of config_str, *p_filename (if it is not a NULL) is
+ * filled with the resulted name of the log file.
```

Comment:
can you pls use doxygen style doc, and note that *p_filename has to be released

Reply:
done